apidef/api_definitions.go

```diff
@@ -377,6 +377,7 @@ type APIDefinition struct {
 		PreserveHostHeader          bool                          `bson:"preserve_host_header" json:"preserve_host_header"`
 		ListenPath                  string                        `bson:"listen_path" json:"listen_path"`
 		TargetURL                   string                        `bson:"target_url" json:"target_url"`
+		TargetURLDontStripSlash     bool                          `bson:"target_url_dont_strip_slash" json:"target_url_dont_strip_slash"`
```

Comment:
`dont` usually not used in naming variables. 
Also, since this is already inside the proxy section, you probably can name it: `disable_strip_prefix` ?

Reply:
i like that better, yes - i was unsure on naming